console-subscriber/src/aggregator/mod.rs

```diff
@@ -75,11 +73,26 @@ pub(crate) struct Aggregator {
     /// Map of resource IDs to resource stats.
     resource_stats: IdData<ResourceStats>,
 
+    /// Map of AsyncOp IDs to AsyncOp static data.
     async_ops: IdData<AsyncOp>,
 
+    /// Map of AsyncOp IDs to AsyncOp stats.
     async_op_stats: IdData<AsyncOpStats>,
 
-    resource_ops: IdData<ResourceOp>,
+    /// *All* PollOp events for AsyncOps on Resources.
+    ///
+    /// This is sent to new clients as part of the initial state.
+    all_poll_ops: ShrinkVec<proto::resources::PollOp>,
+
+    /// *New* PollOp events that whave occured since the last update
+    ///
+    /// This is emptied on every state update.
+    new_poll_ops: ShrinkVec<proto::resources::PollOp>,
+
+    task_ids: Ids,
```

Comment:
Yes, sorry I missed renaming this. I will rename this to just `ids` because this is used now for not just task ids but also resource and async op ids

Reply:
Personally, I would _probably_ give those separate lists of IDs, but we can figure out whether we want to do that or not later...